mapbox/libjava-services/src/test/java/com/mapbox/services/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -168,27 +171,30 @@ public void noProfileProvided() throws ServicesException {
   }
 
   @Test
-  public void requiredCoordinates() throws ServicesException {
+  public void checksCoordinatesAreRequired() throws ServicesException {
     thrown.expect(ServicesException.class);
     thrown.expectMessage(startsWith("You should provide at least two coordinates (from/to)"));
+
     new MapboxDirections.Builder().setAccessToken("pk.XXX").setProfile(DirectionsCriteria.PROFILE_DRIVING).build();
   }
 
   @Test
-  public void testSanity() throws ServicesException, IOException {
+  public void checksSanity() throws ServicesException, IOException {
     MapboxDirections client = new MapboxDirections.Builder()
       .setAccessToken("pk.XXX")
       .setCoordinates(positions)
       .setProfile(DirectionsCriteria.PROFILE_DRIVING)
       .setBaseUrl(mockUrl.toString())
       .build();
+
     Response<DirectionsResponse> response = client.executeCall();
-    assertEquals(response.code(), 200);
-    assertEquals(response.body().getCode(), DirectionsCriteria.RESPONSE_OK);
+
+    assertEquals(200, response.code());
+    assertEquals("Ok", response.body().getCode());
```

Comment:
@Guardiola31337 Thanks for the explanation. While agree with these general goals, I disagree in this particular case. This test really intends to test `response.body().getCode().equals(DirectionsCriteria.RESPONSE_OK)` which is what the developer would type in their code. The fact that `DirectionsCriteria.RESPONSE_OK` has the value `Ok` or `OK` or something else is secondary in this case. In fact, if the server happened to change its response from `Ok` to `OK` and we simply updated the literal here, the test would now pass, but we'd miss updating the value of `DirectionsCriteria.RESPONSE_OK`.